package.json

```diff
@@ -100,5 +101,10 @@
     "stylelint-config-standard": "^34.0.0",
     "stylelint-scss": "^5.2.1",
     "vitest": "^1.4.0"
+  },
+  "overrides": {
+    "s3rver": {
+      "fast-xml-parser": "4.4.1"
```

Comment:
This override is not needed. Was this added by the `npm-force-resolutions`? 

It looks like the `fast-xml-parser` is a dependency for aws-sdk. Checkout this GitHub form https://github.com/aws/aws-sdk-js-v3/issues/6331. It shows how to fix this without doing an override.